src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -141,13 +152,68 @@ your domain. Be sure to include the escaping characters.
 </string>
 ```
 
-### Test for the presence of your native app {: #test-native }
+Finally, publish your updated Android app to the Play store.
+
+### Define the relationship to an installed PWA {: #relationship-web }
+
+{% Aside %}
+**Coming Soon!**
+Expected in Chrome 81, websites can check if their PWA, served from the same
+domain is installed.
+Microsoft is actively working on enabling this API for Edge and we hope to
+see it land around Edge 82.
+{% endAside %}
+
+To define the relationship to an installed PWA, add a `related_applications`
+entry in the web app manifest, set `"platform": "webapp"` and provide
+the full path to the PWAs web app manifest in the `url` property.
+
+```json/3-4
+{
+  …
+  "related_applications": [{
+    "platform": "webapp",
+    "url": "https://app.example.com/manifest.json",
+  }],
+  …
+}
+```
+
+#### PWA served from a different domain
+
+{% Aside %}
+**Coming Soon!**
+Starting in Chrome 82, websites can check if their PWA, served from the same,
+or different domain is installed.
+Microsoft is actively working on enabling this API for Edge and we hope to
+see it land around Edge 82.
+{% endAside %}
 
-Once you've updated your native app and added the appropriate fields to the
-web app manifest, add code to check for the presence of your native
-app to your web app. Calling `navigator.getInstalledRelatedApps()` returns a
-promise that resolves with an array of your apps that are installed on the
-user's device.
+If the PWA you're testing for lives on a different domain, for instance
+`example.com` wants to check if the PWA served from `app.example.com` is
+installed, you'll also need to define the relationship between the two sites.
+
+Add an `assetslink.json` file to the [`/.well-known/`][well-known]
```

Comment:
It has to be called `/.well-known`?

Why should I already know about that?

Reply:
Yes, it needs to be in the `/.well-known/` directory, which itself likely isn't well known. That's why I included the link to the well-known spec.

Reply:
this should be `assetlinks.json`

Reply:
Fixed